packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -1,17 +1,15 @@
 "use client";
 
 import * as React from "react";
+import cx from "clsx";
 
-import KEY_CODE_MAP from "../../common/keyMaps";
-import CardWrapper from "../components/CardWrapper";
-import { useCard } from "../CardContext";
-import SectionHeader from "./components/SectionHeader";
-import SectionContent from "./components/SectionContent";
 import useRandomId from "../../hooks/useRandomId";
 import { ELEMENT_OPTIONS } from "../../Heading/consts";
 import type { Props } from "./types";
+import Header from "../components/Header";
+import Expandable from "./components/Expandable";
 
-const CardSection = ({
+export default function CardSection({
```

Comment:
no besides that I tried to sneak this in since there's so many changes already 😅 

I prefer this because `function` declarations > arrow functions for non-binding function definitions. also, you can one-line `export default` here